time-to-k8s.go

```diff
@@ -273,7 +276,7 @@ func main() {
 
 	c := csv.NewWriter(outputFile)
 
-	c.Write([]string{"name", "args", "platform", "iteration", "time", "version", "exitcode", "error", "command exec (seconds)", "apiserver answering (seconds)", "kubernetes svc (seconds)", "dns svc (seconds)", "app running (seconds)", "dns answering (seconds)", "cpu busy (percent)", "total duration (seconds)"})
```

Comment:
Can you preserve the previous column order? I don't want to break anyones existing workflow.

Reply:
Done.